tools/run_tests/sanity/check_qps_scenario_changes.py

```diff
@@ -27,7 +27,9 @@
 json_run_localhost_bzl = 'test/cpp/qps/json_run_localhost_scenarios.bzl'
 
 if qps_json_driver_bzl in output or json_run_localhost_bzl in output:
-    print('qps benchmark scenarios have been updated, please commit '
+    print('qps benchmark scenarios have been updated, please go '
+          'to //test/cpp/qps directory, run json_run_localhost_scenario_gen.py '
+          'and qps_json_driver_scenario_gen.py scripts, and commit '
```

Comment:
Aren't these scripts run automatically from `tools/buildgen/generate_projects.sh` ? If not, wouldn't that be a better way to run them?

Reply:
they are not run from generate_projects.sh
I was told that there is no pre-commit scripts for gRPC testing, so this is the best solution that I could think of

Reply:
Would it be worthwhile to add to tools/buildgen/generate_projects.sh ? That way it's just one place to fix things no matter what form of project change has happened and then the sanity checks will also notice that projects weren't generated properly.